prow/external-plugins/cherrypicker/main.go

```diff
@@ -68,10 +69,11 @@ func gatherOptions() options {
 	fs.BoolVar(&o.prowAssignments, "use-prow-assignments", true, "Use prow commands to assign cherrypicked PRs.")
 	fs.BoolVar(&o.allowAll, "allow-all", false, "Allow anybody to use automated cherrypicks by skipping GitHub organization membership checks.")
 	fs.BoolVar(&o.issueOnConflict, "create-issue-on-conflict", false, "Create a GitHub issue and assign it to the requestor on cherrypick conflict.")
+	fs.StringVar(&o.labelPrefix, "label-prefix", "", "Set a custom label prefix.")
```

Comment:
You could expose the default prefix as a `const` and make it the default value of this flag, so you don't need the `len() != 0` defaulting logic later.

Reply:
Fixed.